ttnn/cpp/ttnn/operations/full_like/full_like_pybind.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "full_like_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "full_like.hpp"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+#include "ttnn/operations/full_like/device/full_like_device_operation.hpp"
+#include "ttnn/types.hpp"
+
+namespace py = pybind11;
```

Comment:
This should not be here. Already have in hpp.

Reply:
Resolved